dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jaxwsClientSecurity-2.2.feature

```diff
@@ -4,8 +4,8 @@ visibility=private
 IBM-App-ForceRestart: uninstall, \
  install
 IBM-Process-Types: client
-IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.jaxwsClient-2.2))", \
- osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.appSecurityClient-1.0))"
+IBM-Provision-Capability:  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.jaxwsClient-2.2)(osgi.identity=com.ibm.websphere.appserver.jaxwsClient-2.3)))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.appSecurityClient-2.0)(osgi.identity=com.ibm.websphere.appserver.appSecurityClient-1.0)))"
```

Comment:
Since these runtime update is a stepping stone for xmlWsClient-3.0, this change references the 'jaxwsClient-2.3" private feature which we aren't planning on shipping, so this feature change is only for testing during a Personal Build. It will be reverted once it's verified that the runtime changes are working as intended.